src/org/gk/property/SearchDBTypeHelper.java

```diff
@@ -101,35 +103,35 @@ public List mapGKInstanceToRenderable(String type,
                                         instance,
                                         id2Renderable);
         }
-        return new ArrayList(id2Renderable.values());
+        return new ArrayList<Renderable>(id2Renderable.values());
```

Comment:
should: return new ArrayList<>(id2Renderable.values())?

Reply:
Fixed in [b763f88](https://github.com/reactome/CuratorTool/pull/23/commits/b763f888eafed262c4bc2a5c257871707878643a).